src/renderer/src/composables/useInteractiveInput.ts

```
@@ -0,0 +1,508 @@
+
+      // Listen for alternate screen entered (TUI programs like vim, man, git log)
+      const cleanupAlternateScreen = window.api.onAlternateScreenEntered(handleAlternateScreenEntered)
+      cleanupFunctions.push(cleanupAlternateScreen)
+
+      console.log('[useCommandInteraction] IPC listeners registered')
+    }
+  })
+
+  // Cleanup on unmount
+  onUnmounted(() => {
+    cleanupFunctions.forEach((fn) => fn())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)